src/fusion_impl.rs

```diff
@@ -15,7 +15,7 @@ impl Fusion {
             hard_iron_offset: FusionVector::zero(),
             ahrs,
             offset: FusionGyrOffset::new(sample_rate),
-            last_dt: 0f32,
+            last_time_stamp: 0f32,
```

Comment:
Lets change time_stamp & last_time_stamp to single word, timestamp & last_timestamp accordingly

Reply:
Done